source/sink-connector/configuration-properties/metrics.txt

```diff
@@ -0,0 +1,281 @@
+.. _sink-configuration-metrics:
+
+==========================
+Sink Connector JMX Metrics
+==========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+.. _sink-configuration-metrics-description-start:
+
+Use the metrics in the following table to monitor the behavior of your sink
+connector through {+jmx-long+} (JMX).
+
+.. _sink-configuration-metrics-description-end:
+
+.. include:: /includes/sink-config-link.rst
+
+.. _sink-connector-metrics:
+
+Metrics
+-------
+
+.. _sink-configuration-metrics-table-start:
+
+.. tip:: JMX Attributes
+
+   {+jmx-hover+} represents an individual metric as an attribute of an ``MBean``.
+   To learn more about attributes and ``MBeans``, see the
+   `Standard MBeans Tutorial <https://docs.oracle.com/javase/tutorial/jmx/mbeans/standard.html>`__
+   from Oracle.
+
+.. list-table::
```

Comment:
The content of the JMX Metrics table has already been looked over by a DevED-DBX team member and has been reviewed by DBX-JVM team. Please focus review of this table on formatting.